common/language.h

```diff
@@ -76,6 +77,8 @@ enum Language {
 	TR_TUR,
 	UA_UKR,
 	SR_SER,
+	CA_ESP,
```

Comment:
> It would be nice to sort these alphabetically too.

Unless there are things that depend on the exact numerical values. Hopefully nothing does, though. I know create_access used to, but that appears to have been fixed now?